hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -119,37 +127,25 @@
         _ (.setProperty config "hazelcast.client.heartbeat.interval" "1000")
         _ (.setProperty config "hazelcast.client.heartbeat.timeout" "5000")
         _ (.setProperty config "hazelcast.client.invocation.timeout.seconds" "5")
-        _ (.setProperty config "hazelcast.heartbeat.interval.seconds" "1")
-        _ (.setProperty config "hazelcast.master.confirmation.interval.seconds" "1")
-        _ (.setProperty config "hazelcast.max.no.heartbeat.seconds" "5")
-        _ (.setProperty config "hazelcast.max.no.master.confirmation.seconds" "10")
-        _ (.setProperty config "hazelcast.operation.call.timeout.millis" "5000")
```

Comment:
Just as a heads-up, the nemeses were designed with particular timings in mind, which depended (in previous versions) on these parameters; you'll want to make sure you've explored the parameter space and that your nemeses are actually triggering the kinds of failover events you'd like to test.

Reply:
When these parameters are set via `ClientConfig`, they are not respected in the server side. So we set them in `hazelcast_server.clj`.

Reply:
Ah, very good!